doc: known_issues: Add known issue to SoftDevice Controller section (#13483)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

.. rst-class:: v2-5-1 v2-5-0 v2-4-2 v2-4-1 v2-4-0

> DRGN-21020: The continuous extended scanner can stop generating advertising reports

Reviewer: ```suggestion
DRGN-21020: The continuous extended scanner sometimes stops generating advertising reports
```

>   This can happen when the controller is running extended cooperative scanner together with other activities (e.g., advertising or connection)
>   and is receiving data in an extended advertising event with ``AUX_CHAIN_IND`` used.

Reviewer: ```suggestion
  This can happen when the controller is running an extended cooperative scanner together with other activities, such as advertising or connection, while receiving data in an extended advertising event that uses ``AUX_CHAIN_IND``.
```
